Gallery-Server/src/main/java/edu/yu/cs/gallery/GalleryResource.java

```diff
@@ -45,31 +47,31 @@ public class GalleryResource {
     String hubURL;
 
     @GET
-        public Response getOnServer() {
-        return Response.status(Status.OK).entity(gr.findAll().firstResult()).build();
-    }
-
-    @GET
-    @Path("batch")
-    public Response getOnServer(@QueryParam("gallery") long[] galleries, @Context UriInfo uriInfo) throws URISyntaxException {
-        return utility.redirect(galleries, uriInfo);
+    public Response getOnServer() {
+    return Response.status(Status.OK).entity(gr.findAll().firstResult()).build();
     }
 
     @GET
     @Path("{id}")
     public Response getById(@PathParam("id") Long id, @Context UriInfo uriInfo) throws URISyntaxException {
         if (utility.gallery == null || id != utility.gallery.id) {
-            return utility.redirect(id, uriInfo);
+            return utility.temporaryRedirect(id, uriInfo);
         }
-        return Response.status(Status.FOUND).entity(gr.findById(id)).build();
+        return Response.status(Status.OK).entity(gr.findById(id)).build();
     }
     
+    @POST
```

Comment:
should this still be here?

Reply:
Nope. We had it for testing purposes and forgot to delete it